pint/compat.py

```diff
@@ -11,8 +11,14 @@
 import tokenize
 from decimal import Decimal
 from io import BytesIO
+from math import exp as math_exp
+from math import log as math_log
 from numbers import Number
 
+# Defines Logarithm and Exponential for Logarithmic Converter
+from numpy import exp as np_exp
```

Comment:
This is not a good idea, because numpy might not be installed

Reply:
I agree, it's not great. But I was getting a warning of unused libraries by flake8.
Any idea of how to get around this issue?

Reply:
OK, I found a way to solve this: you can find it [at this commit](https://github.com/hgrecco/pint/pull/1024/commits/fcab800d3e6267fa6f2c13f3dfa4e0cb192827c4)